extensions/README.md

```diff
@@ -47,22 +47,48 @@ types. This is done via three distinct extension mechanisms:
 
 Map extensions allow extending CoRIM maps with additional keys, effectively
 defining new fields for the corresponding structures. In the code base, these
-can be identified by the embedded `Extensions` struct. These are
-
-- `comid.Comid`
-- `comid.Entity`
-- `comid.FlagsMap`
-- `comid.Mval`
-- `comid.Triples`
-- `corim.Entity`
-- `corim.Signer`
-- `corim.UnsignedCorim`
+can be identified by the embedded `Extensions` struct. Each extensible type has
+a corresponding `extensions.Point`. These are:
+
+| type                  | extension point                                               |
+| --------------------- | ------------------------------------------------------------- |
+| `comid.Comid`         | `comid.ExtComid`                                              |
+| `comid.Entity`        | `comid.ExtEntity`                                             |
+| `comid.FlagsMap`      | `comid.ExtReferenceValueFlags`, `comid.ExtEndorsedValueFlags` |
+| `comid.Mval`          | `comid.ExtReferenceValue`, `comid.ExtEndorsedValue`           |
+| `comid.Triples`       | `comid.ExtTriples`                                            |
+| `corim.Entity`        | `corim.ExtEntity`                                             |
+| `corim.Signer`        | `corim.ExtSigner`                                             |
+| `corim.UnsignedCorim` | `corim.ExtUnsignedCorim`                                      |
+
+Note that `comid.Mval` and `comid.FlagsMap` are used for both reference and
+endorsed values, which may be extended separately. This is why there are two
+extension points associated with each. Additionally, `comid.ExtMval` and
+`comid.ExtFlags` also exist for when you want to register extensions with a
```

Comment:
```suggestion
`comid.ExtFlags` also exist when you want to register extensions with a
```

Reply:
"for when" is the correct construct here as it implies purpose (i.e. they don't just exist at the time you want to extend `Mval` and `Measurement`, but _for_ that time).